demo/src/lib/gtag.js

```diff
@@ -0,0 +1,24 @@
+
+// eslint-disable-next-line no-undef
```

Comment:
It's not common to suppress lint warnings, are you sure it cannot be avoided?

Reply:
I did some more research into this - I usually have an empty [dependency array](https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects) to make it very explicit that the code runs exactly once but it seems this has fallen out of favor.

The history object should be safe to pass into the array as it should never change ensuring the code runs exactly once. Some testing using the Debug Mode seems to confirm this.

Reply:
Whoops I originally though this referring to a different part of the PR. Ignore any previous notifications on this. I looked into a bit more and adjusted the eslint config to allow access to process. Thanks!